mkdocs.yml

```diff
@@ -6,17 +6,18 @@ nav:
       - Common principles: common-principles.md
       - Modality agnostic files: modality-agnostic-files.md
       - Modality specific files:
-        - Magnetic Resonance Imaging: modality-specific-files/magnetic-resonance-imaging-data.md
-        - Magnetoencephalography: modality-specific-files/magnetoencephalography.md
-        - Electroencephalography: modality-specific-files/electroencephalography.md
-        - Intracranial Electroencephalography: modality-specific-files/intracranial-electroencephalography.md
-        - Task events: modality-specific-files/task-events.md
-        - Physiological and other continuous recordings: modality-specific-files/physiological-and-other-continuous-recordings.md
         - Behavioral experiments (with no neural recordings): modality-specific-files/behavioral-experiments.md
+        - Electroencephalography: modality-specific-files/electroencephalography.md
+        - Eye Tracking: modality-specific-files/eye-tracking.md
         - Genetic Descriptor: modality-specific-files/genetic-descriptor.md
-        - Positron Emission Tomography: modality-specific-files/positron-emission-tomography.md
+        - Intracranial Electroencephalography: modality-specific-files/intracranial-electroencephalography.md
+        - Magnetic Resonance Imaging: modality-specific-files/magnetic-resonance-imaging-data.md
+        - Magnetoencephalography: modality-specific-files/magnetoencephalography.md
         - Microscopy: modality-specific-files/microscopy.md
         - Near-Infrared Spectroscopy: modality-specific-files/near-infrared-spectroscopy.md
+        - Physiological and other continuous recordings: modality-specific-files/physiological-and-other-continuous-recordings.md
+        - Positron Emission Tomography: modality-specific-files/positron-emission-tomography.md
+        - Task events: modality-specific-files/task-events.md
```

Comment:
+1 on the sorting, but I think we better do it as a separate PR to keep this PR specific to the BEP.  I prepped that PR #1366 -- not sure if that would not cause a conflict if merged.